### PR TITLE
Feature/#1615 content headers

### PR DESCRIPTION
**Pull request by vgenev**
added config for resource API version

### CODE REVIEW

**Review by lewisdaly — CHANGES_REQUESTED** [review]
Thanks @vgenev - just a couple minor points, and it would be nice to see some unit tests.

File: src/config.js [hunk 1]
```diff
@@ -131,5 +131,7 @@ module.exports = {
     outboundErrorStatusCodeExtensionKey: env.get('OUTBOUND_ERROR_STATUSCODE_EXTENSION_KEY').asString(),
 
     proxyConfig: env.get('PROXY_CONFIG_PATH').asYamlConfig(),
-    reserveNotification: env.get('RESERVE_NOTIFICATION').default('false').asBool()
+    reserveNotification: env.get('RESERVE_NOTIFICATION').default('false').asBool(),
+    // resourcesVersion config should be string in format: "resouceOneName=1.0,resourceTwoName=1.1"
+    resourcesVersion: env.get('RESOURCES_VERSIONS').default('').asString()
```
[the remark this answers is not in the document]
Comment (vgenev) [reply src/config.js:136]: sure!

File: src/config.js [hunk 2]
```diff
@@ -14,17 +14,46 @@ require('dotenv').config();
...
+            .reduce((p, c) => {
+                resourceVersionMap[p] = {
+                    contentVersion: c,
+                    acceptVersion: c.split('.')[0],
+                };
+            }));
+    return resourceVersionMap;
+}
+    
+function validateResourceVersions (resourceString) {
```
Comment (lewisdaly) [inline src/config.js:42]: I would rename this to `parseResourceVersions`, since it's both validating and parsing.

File: src/config.js [hunk 3]
```diff
@@ -14,17 +14,46 @@ require('dotenv').config();
...
+    if (!resourceString) return '';
```
Comment (lewisdaly) [inline src/config.js:43]: Is this what we want to do? Won't this cause other downstream breaks in the `sdk-standard-components`? Perhaps we should return an empty map instead.

File: package-lock.json [hunk 4]
```diff
@@ -0,0 +1,4178 @@
+{
```
Comment (lewisdaly) [inline package-lock.json:1]: This file shouldn't be here.

**Review by lewisdaly — CHANGES_REQUESTED** [review]
Just a few minor notes and thoughts.

File: src/config.js (hunk 3 above)
Comment (vgenev) [reply src/config.js:43]: technically it won't break but makes much more sense to return empty map, yeah

File: package-lock.json (hunk 4 above)
Comment (vgenev) [reply package-lock.json:1]: that ba*d.. :>

File: src/test/unit/config.test.js [hunk 5]
```diff
@@ -93,4 +93,23 @@ describe('config', () => {
...
+                acceptVersion: '1',
+                contentVersion: '1.1',
+            },
+
+        };
+        process.env.RESOURCE_VERSIONS = 'resourceOneName=1.0,resourceTwoName=1.1';
+        const config = require('../../config');
+        expect(config.resourceVersions).toEqual(resourceVersions);
+    });
+
```
[the remark this answers is not in the document]
Comment (vgenev) [reply src/test/unit/config.test.js:114]: I was just following the pattern.. but yeah..

**Review by lewisdaly — APPROVED** [review]
Looks good!

**Pull request merged**